### ISSUE TITLE
Can we have a getting started documentation?

### ISSUE THREAD

**Issue by arpansolanki** [spinnaker/swabbie#110]
It will be nice to have a getting started documentation on steps of how to install and configure swabbie.

**Comment by emjburns** [comment]
@arpansolanki you're right! When we feel Swabbie is stable and ready for production we will document this.

**Comment by EleanorRigby** [comment]
@emjburns: I would like to still go ahead and test it for our systems and give feedback. If you can provide a short doc on deploying and getting it up with fewest configurations, that would be awesome

**Comment by supersundar** [comment]
I installed successfully but i need steps such as what changes/where i should make changes to configure to use the redis cluster etc. and how to run
